src/views/FAQ.vue

```diff
@@ -66,6 +74,38 @@ export default {
           "Na klar! Schreib uns eine Mail über das Feedback-Formular. Wir freuen uns über Ideen, Benutzerhinweise - oder auch über das Angebot, am Projekt mitzuarbeiten."
       }
     ]
-  })
+  }),
+  async created() {
+    try {
+      await Promise.all([this.$store.dispatch("faq/queryFaq")]);
+    } catch (error) {
+      log(error);
+      this.error = true;
+    } finally {
+      this.loading = false;
+    }
+  },
+  computed: {
+    ...mapGetters({
+      loading: "faq/loading",
+      faqs: "faq/faqs"
+    })
+  },
+  methods: {
+    question(faq) {
+      console.log(this.$i18n.locale);
+      if (this.$i18n.locale === "de") {
```

Comment:
Ich rechne zwar nicht unbedingt damit, dass wir in Zukunft noch großartig weitere Sprachen hinzunehmen.

Aber irgendwie bin ich noch nicht überzeugt, ob diese binäre Unterscheidung die bestmögliche ist.
Abgesehen davon, dass das auch mit einem ternären Operator ginge (😉) wäre eine dynamische Lösung auch schön, also ein Prüfen nach dem locale-Kürzel, Auswahl des korrekten Feldes und ein Defaultwert (de) falls es keinen match gibt.

Reply:
ABER dann müsste auch die Logik des Endpoints vermutlich etwas umgestrickt werden. 
Da die paar FAQ nicht viel Payload ausmachen, ist es okay, wenn wir immer beides laden. Sonst hätte ich gesagt, dass die Locale beim Request mitgesendet wird und das Backend nur die gewünschte Sprache ausspuckt. Dann müssten aber konsequenterweise auch die FAQ bei einem Sprachwechsel neu geladen werden.

Ich sehe im Moment aber keine Indikation dafür - wir können das so lassen. Sollten eines Tages noch mehr Sprachen dazukommen, können wir das immer noch anfassen.

Reply:
Habs mal ternär umgebaut, sieht schicker aus, hast Recht.

Wie ich das dynamisch hinbekomme wüsste ich jetzt nicht, aber als default fallback finde ich ehrlich gesagt Englisch am praktischsten. Nicht das sich jmd mit lokalen spanisch Einstellungen anmeldet und alles auf deutsch bekommt...

Btw ist der Hinweis zum Handbuch auch noch deutsch. Soll ich das anpassen analog zu den Fragen und Antworten Methoden?

Reply:
Ahja, das sollte in die locale und automatisch übersetzt werden.